mkt/api/tests/__init__.py

```diff
@@ -19,7 +21,8 @@ def _allowed_verbs(self, url, allowed):
             if verb in allowed:
                 continue
             try:
-                res = getattr(self.client, verb)(url)
+                with transaction.atomic():
```

Comment:
Discussed on IRC and it makes sense, DRF now rolls back the transaction in the exception handler if `ATOMIC_REQUESTS` is `True` (or inside a manual transaction), and we want that.